papers/H0_I/Analysis/CRACO/Cloud/run_nautilus_test4.py

```diff
@@ -0,0 +1,96 @@
+""" Run a Nautilus test """
+
```

Comment:
Can you add notes on what is different between test 1, test2, test4 etc? test2 states "long enough to monitor CPU and RAM", which makes sense, but I don't see the uniqueness here.